File: java/src/main/java/com/dddeurope/niuz/events/ArticlePublished.java
package com.dddeurope.niuz.events;

public class ArticlePublished {
    private String headline;
    private String author;

    public ArticlePublished(String headline, String author) {
        this.headline = headline;
        this.author = author;
    }

    public String getHeadline() {
        return headline;
    }

    public String getAuthor() {
        return author;
    }
}

File: dotnet/niuz.application/events/ArticlePublished.cs
namespace niuz.application.events
{
    public class ArticlePublished
    {
        public ArticlePublished(string authorId, string authorName, string headline)
        {
            Headline = headline;
            AuthorName = authorName;
            AuthorId = authorId;
        }

        public string AuthorId { get; }
        public string AuthorName { get; }
        public string Headline { get; }
    }
}